### PR TITLE
remove decimals_as_float64 from orc benchmarks

### PR DESCRIPTION
**Pull request by cwharris**
`decimals_as_float64` was recently removed in https://github.com/rapidsai/cudf/pull/7970. This PR fix benchmark builds by removing it's usage from benchmarks.

### CODE REVIEW

**Review by galipremsagar — APPROVED** [review]

**Review by codereport — APPROVED** [review]
:+1:

**Review by devavret — CHANGES_REQUESTED** [review]
Can drop the MSB from these then? https://github.com/rapidsai/cudf/blob/41af1581f5668a4ec5f8f58acd4dcb6fd29495d7/cpp/benchmarks/io/orc/orc_reader_benchmark.cpp#L168
https://github.com/rapidsai/cudf/blob/41af1581f5668a4ec5f8f58acd4dcb6fd29495d7/cpp/benchmarks/io/orc/orc_reader_benchmark.cpp#L179
https://github.com/rapidsai/cudf/blob/41af1581f5668a4ec5f8f58acd4dcb6fd29495d7/cpp/benchmarks/io/orc/orc_reader_benchmark.cpp#L190

File: cpp/benchmarks/io/orc/orc_reader_benchmark.cpp [hunk 1]
```diff
@@ -107,8 +106,7 @@ void BM_orc_read_varying_options(benchmark::State& state)
       .columns(cols_to_read)
       .use_index(use_index)
       .use_np_dtypes(use_np_dtypes)
-      .timestamp_type(ts_type)
-      .decimals_as_float64(dec_as_float);
+      .timestamp_type(ts_type);
 
   auto const num_stripes              = data_size / (64 << 20);
   cudf::size_type const chunk_row_cnt = view.num_rows() / num_chunks;
```
Comment (vuule) [inline cpp/benchmarks/io/orc/orc_reader_benchmark.cpp:112]: Can't leave a comment there, but line 190 should be modified to only vary the two remaining options:
`{0b11, 0b10, 0b01},  // `true` is default for each boolean parameter here`

**Review by vuule — CHANGES_REQUESTED** [review]
missed one more line that needs to change

**Review by vuule — APPROVED** [review]

**Review by devavret — APPROVED** [review]

**Comment by vuule** [comment]
@gpucibot merge

**Pull request merged**